docs/source/upgrade-guide/index.md

```diff
@@ -521,6 +521,34 @@ You'll have to add a script in your `package.json` file called `prepare`:
 
 After execute it, `husky` will install itself in the `.husky` folder of your project. Then you need to create the default hook scripts in `.husky` that you want to execute. You can copy over the Volto ones (take a look in Volto's `.husky` folder).
 
+### Better defaults handling
```

Comment:
I just made a PR to 16.x.x that cleans up the upgrade guide. I'm not sure how to handle this. Let's chat.

https://github.com/plone/volto/pull/3930